admin-console/src/test/java/com/agiletec/apsadmin/system/entity/TestEntityManagersAction.java

```diff
@@ -230,30 +222,25 @@ public void testSaveNewAttribute() throws Throwable  {
 			assertEquals(1, fieldErrors.size());
 			assertTrue(fieldErrors.containsKey("attributeName"));
 			//wrong length
-			attributeName="right_name";
+			attributeName="rightName";
 			result = this.executeSaveAttribute("admin", attributeName, attributeTypeCode, entityTypeCode, strutsAction, 3, 5, entityManagerName);
 			assertEquals(Action.INPUT, result);
+			action = (EntityAttributeConfigAction) getAction();
 			fieldErrors = action.getFieldErrors();
 			assertEquals(1, fieldErrors.size());
-			assertTrue(fieldErrors.containsKey("minLength"));
+			assertTrue(fieldErrors.containsKey("maxLength"));
 			//insert ok
-			attributeName="right_name";
-			result = this.executeSaveAttribute("admin", attributeName, attributeTypeCode, entityTypeCode, strutsAction, 5, 3, entityManagerName);
+			attributeName="rightName";
+			result = this.executeSaveAttribute("admin", attributeName, attributeTypeCode, entityTypeCode, strutsAction, 10, 3, entityManagerName);
 			assertEquals(Action.SUCCESS, result);
-			IApsEntity entity = entityManager.getEntityPrototype(entityTypeCode);
-			assertTrue(entity.getAttributeMap().containsKey("right_name"));
-			//duplicate attribute name
-			attributeName="right_name";
-			result = this.executeSaveAttribute("admin", attributeName, attributeTypeCode, entityTypeCode, strutsAction, null, null, entityManagerName);
-			assertEquals(Action.INPUT, result);
+			action = (EntityAttributeConfigAction) getAction();
 			fieldErrors = action.getFieldErrors();
-			assertEquals(1, fieldErrors.size());
-			assertTrue(fieldErrors.containsKey("attributeName"));
+			assertTrue(fieldErrors.isEmpty());
 		} catch (Throwable t) {
```

Comment:
Because of this catch in Throwable, and the empty catch block, this test was raising an error in one of the first asserts and after that it was aborting because the catch was swallowing the assert error.